cvat/apps/engine/static/engine/js/userConfig.js

```diff
@@ -132,6 +132,12 @@ class Config {
                 description: "focus to 'go to frame' element"
             },
 
+            set_key_frame: {
+                value: "a",
+                view_value: "A",
+                description: "Set current frame as key frame"
```

Comment:
Action is actually toggle (i.e. switch to different value on -> off, off -> on) and description should reflect it